### PR TITLE
Rerun 0.18.0

### PR DESCRIPTION
**Pull request by teh-cmc**
TODO:
- [x] This branch removes `examples-ocr` because it broke CI build -- make sure to revert that
- [ ] This branch adds `binaryen` to `wheel-test` -- make sure to revert that (?)
- [x] handle link todos before putting up release notes
- [x] include https://github.com/rerun-io/rerun/pull/7184
- [x] include https://github.com/rerun-io/rerun/pull/7183
- [x] include https://github.com/rerun-io/rerun/pull/7185

### Next steps
- Test the release
- If this is an 'alpha' release, you can just merge the pull request.
- Otherwise:
  - For any added commits, run the release workflow in 'rc' mode again
  - After testing, _ensure that this PR is mergeable to `main`_, then run the release workflow in 'release' mode
  - Once the final release workflow finishes it will create a GitHub release for you. Then:
    - [x] Sanity check the build artifacts:
      - [x] pip install: does it install and run?
      - [x] cargo install of cli tool: does it install and run?
      - [x] C++ SDK zip: does it contain rerun_c for all platforms?
    - [x] Populate the release with the changelog and a nice header video/picture, check `Set as latest release`, then click `Publish release`.
    - [x] Update the [google colab notebooks](https://drive.google.com/drive/folders/0AC0q24MFKh3fUk9PVA) to install this version and re-execute the notebook.

A few hours after the GitHub release is created, `regro-cf-autotick-bot` will create a
[conda feedstock PR](https://github.com/conda-forge/rerun-sdk-feedstock/pulls).
Make sure Jeremy is on top of it!

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)
* [x] I've included a screenshot or gif (if applicable)
* [x] I have tested the web demo (if applicable):
  * Using examples from latest `main` build: [rerun.io/viewer](https://rerun.io/viewer/pr/7146?manifest_url=https://app.rerun.io/version/main/examples_manifest.json)
  * Using full set of examples from `nightly` build: [rerun.io/viewer](https://rerun.io/viewer/pr/7146?manifest_url=https://app.rerun.io/version/nightly/examples_manifest.json)
* [x] The PR title and labels are set such as to maximize their usefulness for the next release's CHANGELOG
* [x] If applicable, add a new check to the [release checklist](https://github.com/rerun-io/rerun/blob/main/tests/python/release_checklist)!
* [x] If have noted any breaking changes to the log API in `CHANGELOG.md` and the migration guide

- [PR Build Summary](https://build.rerun.io/pr/7146)
- [Recent benchmark results](https://build.rerun.io/graphs/crates.html)
- [Wasm size tracking](https://build.rerun.io/graphs/sizes.html)

To run all checks from `main`, comment on the PR with `@rerun-bot full-check`.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +149,159 @@
 
 🧳 Migration guide: http://rerun.io/docs/reference/migration/migration-0-18?speculative-link
 
+### 🔎 Details
+
+#### 🪵 Log API
+- Add `Ellipsoids` archetype [#6853](https://github.com/rerun-io/rerun/pull/6853) (thanks [@kpreid](https://github.com/kpreid)!)
```
Comment (abey79) [inline CHANGELOG.md:155]:
```suggestion
- Add `Ellipsoids3D` archetype [#6853](https://github.com/rerun-io/rerun/pull/6853) (thanks [@kpreid](https://github.com/kpreid)!)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,146 @@
...
+To help improve efficiency for completed recordings, Rerun 0.18 introduces some new commands for working with rrd files.
+
+Multiple files can be merged, whole entity paths can be dropped, and chunks can be compacted.
+
+You can read more about it [in the new CLI reference manual](https://rerun.io/docs/reference/cli), but to give a sense of how it works the below example merges all recordings in a folder and runs chunk compaction using the `max-rows` and `max-bytes` settings:
+```sh
+rerun rrd compact --max-rows 4096 --max-bytes=1048576 /my/recordings/*.rrd > output.rrd
+```
+
+### ⚠️  Breaking changes
```
Comment (abey79) [inline CHANGELOG.md:143]: Should we include the regression on "plot hole" somewhere here, or in another "Known issues" section?
- https://github.com/rerun-io/rerun/pull/6586

Comment (teh-cmc) [reply CHANGELOG.md:143]: Oh nice catch, I thought we shipped that in 0.17.

I'll add this to the breaking change section... unless I fix it today?!

Comment (teh-cmc) [reply CHANGELOG.md:143]: https://github.com/rerun-io/rerun/pull/7184

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,8 +1,146 @@
...
 * `mesh_material: Material` has been renamed to `albedo_factor: AlbedoFactor` [#6841](https://github.com/rerun-io/rerun/pull/6841)
```
Comment (emilk) [inline CHANGELOG.md:144]: ```suggestion`
* `.rrd` files from older versions won't load correctly in Rerun 0.18
* `mesh_material: Material` has been renamed to `albedo_factor: AlbedoFactor` [#6841](https://github.com/rerun-io/rerun/pull/6841)
```

**Pull request merged**